kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/status/NMStatusConverter.java

```diff
@@ -376,7 +377,11 @@ private static List<WifiAccessPoint> wifiAccessPointConvert(List<Properties> nmA
         List<WifiAccessPoint> kuraAccessPoints = new ArrayList<>();
 
         for (Properties prop : nmAccessPoints) {
-            kuraAccessPoints.add(wifiAccessPointConvert(prop));
+            try {
+                kuraAccessPoints.add(wifiAccessPointConvert(prop));
+            } catch (Exception e) {
```

Comment:
Does exist a specific exception? Or we cannot predict the error thrown, so it's better to catch them all?

Reply:
i wuold catch all, in any case wa want to skip that access point